src/java/org/apache/cassandra/db/compaction/Scrubber.java

```diff
@@ -381,7 +381,7 @@ private boolean tryAppend(DecoratedKey prevKey, DecoratedKey key, SSTableRewrite
         OrderCheckerIterator sstableIterator = new OrderCheckerIterator(getIterator(key),
                                                                         realm.metadata().comparator);
 
-        try (UnfilteredRowIterator iterator = withValidation(sstableIterator, dataFile.getPath()))
+        try (UnfilteredRowIterator iterator = withValidation(sstableIterator, dataFile.getPath().toString()))
```

Comment:
q: Is `toString` valid in this context - should't we use URI?

Reply:
I think it returns URI in string format, and the value is only used for exception.

Reply:
I'm afraid it does not return URI, but you are right, since it is used only for exception it should be ok